ephypype/commands/tests/test_cli.py

```diff
@@ -9,6 +9,9 @@
 from ephypype.commands import neuropycon
 from click.testing import CliRunner
 
+import matplotlib
+matplotlib.use('Agg')  # for testing don't use X server
```

Comment:
can you put it at the top before any import?

Reply:
also, in MNE all the `matplotlib` imports are nested inside functions, not globally, precisely to avoid these kinds of bugs ...